csfieldguide/chapters/management/commands/_ChaptersLoader.py

```diff
@@ -34,51 +37,62 @@ def load(self):
             MissingRequiredFieldError: When a config (yaml) file is missing a required
                 field.
         """
-        chapter_number = self.chapter_structure.get("chapter-number", None)
-        if chapter_number is None:
-            raise MissingRequiredFieldError(
-                self.structure_file_path,
-                ["sections", "chapter-number", "title"],
-                "Chapter number"
-            )
+        chapter_introduction = self.convert_md_file(
+            os.path.join(
+                self.chapter_path,
+                "{}.md".format(self.chapter_slug)
+            ),
+            self.chapter_structure_file_path,
+        )
 
-        chapter_title = self.chapter_structure.get("title", None)
-        if chapter_title is None:
+        chapter_icon = self.chapter_structure.get("icon", None)
+        if chapter_icon is None:
             raise MissingRequiredFieldError(
-                self.structure_file_path,
-                ["sections", "chapter-number", "title"],
-                "Chapter title"
+                self.chapter_structure_file_path,
+                self.required_fields,
+                "Chapter"
             )
 
-        chapter_icon = self.chapter_structure.get("icon", None)
+        chapter_other_resources_html = None
+        if "other-resources" in self.chapter_structure:
+            chapter_other_resources_file = self.chapter_structure.get("other-resources", None)
+            if chapter_other_resources_file is not None:
+                other_resources_content = self.convert_md_file(
+                    os.path.join(
+                        self.chapter_path,
+                        chapter_other_resources_file,
+                    ),
+                    self.chapter_structure_file_path,
+                )
+                chapter_other_resources_html = other_resources_content.html_string
 
         # Create chapter object and save to the db
         chapter = Chapter(
             slug=self.chapter_slug,
-            name=chapter_title,
-            number=chapter_number,
-            other_resources=None,
+            name=chapter_introduction.title,
+            number=self.chapter_number,
+            introduction=chapter_introduction.html_string,
+            other_resources=chapter_other_resources_html,
             icon=chapter_icon
         )
         chapter.save()
 
-        chapter_sections = self.chapter_structure.get("sections", None)
-        if chapter_sections is None:
+        self.log("Added Chapter: {}".format(chapter.name))
+
+        sections_yaml = self.chapter_structure.get("sections", None)
+        if sections_yaml is None:
             raise MissingRequiredFieldError(
-                self.structure_file_path,
-                ["sections", "chapter-number", "title"],
-                "Chapter sections"
+                self.chapter_structure_file_path,
+                self.required_fields,
+                "Chapter"
```

Comment:
I believe this is the wrong term here?

Reply:
It looks right to me...
This throws the following error when a chapter yaml file is missing the `sections` key.
```
A Chapter requires the following field:
  - ['sections', 'icon']

For the missing field:
  - Is the field name spelt correctly?
  - Does the field have the correct value?
```